frontend/admin/src/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -29,7 +29,6 @@ import {
   getOperationalRequirement,
 } from "@common/constants/localizedConstants";
 import { errorMessages } from "@common/messages";
-import { phoneNumberRegex } from "@common/constants/regularExpressions";
```

Comment:
I wonder if we should remove this regex from the common file if it's not going to be used anywhere?